types/test/models/User.ts

```diff
@@ -26,7 +26,7 @@ export interface UserAttributes {
  */
 export interface UserCreationAttributes extends Optional<UserAttributes, 'id' | 'username' | 'lastName' | 'groupId'> {}
 
-export class User extends Model<UserAttributes, UserCreationAttributes> implements UserAttributes {
+export class User extends Model<UserAttributes, UserCreationAttributes> {
```

Comment:
I removed this as it seemed not to have a bearing on the tests, but happy to reverse this change if its important for some reason I do not understand. 

```suggestion
export class User extends Model<UserAttributes, UserCreationAttributes> implements UserAttributes {
```

Reply:
Ah - I was being silly - of course this is required - its here to ensure that User has properties assigned to it correctly 🤦